libs/ml/tests/unit/model/sequential_test.cpp

```diff
@@ -112,7 +112,7 @@ TYPED_TEST(SequentialModelTest, adagrad_sequential)
 {
   using DataType = typename TypeParam::Type;
   ASSERT_TRUE(sequential_details::RunTest<TypeParam>(fetch::ml::OptimiserType::ADAGRAD,
-                                                     DataType{1e-4f}, DataType{0.03f}, 400));
+                                                     DataType{1e-4f}, DataType{0.04f}, 400));
```

Comment:
Scientific notation

Reply:
ML tests will be defloated in a separate PR